model/patch/cli_intent.go

```diff
@@ -205,22 +222,18 @@ func NewCliIntent(user, project, baseHash, module, patchContent, description str
 	return &cliIntent{
 		DocumentID:    mgobson.NewObjectId().Hex(),
 		IntentType:    CliIntentType,
-		PatchContent:  patchContent,
-		Description:   description,
-		BuildVariants: variants,
-		Tasks:         tasks,
-		SyncAtEndOpts: SyncAtEndOptions{
-			BuildVariants: syncBVs,
-			Tasks:         syncTasks,
-			Statuses:      syncStatuses,
-			Timeout:       syncTimeout,
-		},
-		User:      user,
-		ProjectID: project,
-		BaseHash:  baseHash,
-		Finalize:  finalize,
-		Module:    module,
-		Alias:     alias,
+		PatchContent:  params.PatchContent,
+		Description:   params.Description,
+		BuildVariants: params.Variants,
+		Tasks:         params.Tasks,
+		SyncAtEndOpts: params.SyncParams,
+		User:          params.User,
+		ProjectID:     params.Project,
+		BaseHash:      params.BaseGitHash,
+		Finalize:      params.Finalize,
+		Module:        params.Module,
+		Alias:         params.Alias,
+		IsMbox:        len(params.PatchContent) == 0 || IsMailboxDiff(params.PatchContent),
```

Comment:
Since PatchContent is already saved, do we need an IsMbox field, or when you check this can you call a function that checks this condition?